swagger.yaml

```diff
@@ -25,6 +25,8 @@ tags:
     description: Manifests for customizing a cluster installation.
   - name: versions
     description: Information regarding versions.
+  - name: operators
```

Comment:
The linter should require alphabetical order here?

Reply:
the linter didn't fail but I fixed it